packages/api/portfolios/environments/createEnvironment.ts

```diff
@@ -96,11 +89,29 @@ export async function baseHandler(
   return new ApiSuccessResponse<Array<IEnvironmentCreate>>(insertedEnvironments, SuccessStatusCode.CREATED);
 }
 
+// a workaround for ensuring validation does not give a false failure due to 'allOf'
+// also a band-aid for ensuring BaseObject properties (e.g., id) are not accepted
+// when sent in the request body
+const { description, type, additionalProperties, required } = internalSchema.Environment;
+const { administrators, contributors, readOnlyOperators } = internalSchema.Environment.allOf[1].properties;
```

Comment:
Would this be clearer if it referred directly to `internalSchema.AppEnvAccess.properties`?